README.rst

```diff
@@ -113,18 +113,34 @@ Example
 
 When installing Qt SDK 6.2.0 for Windows.
 
-Check the options that can be used with the `list-qt` subcommand, and query available architectures
+Check the options that can be used with the ``list-qt`` subcommand, and query available architectures
 
 .. code-block:: console
 
     aqt list-qt windows desktop --arch 6.2.0
 
-Then you may get candidates: `win64_mingw81 win64_msvc2019_64 win64_msvc2019_arm64`.
+Then you may get candidates: ``win64_mingw81 win64_msvc2019_64 win64_msvc2019_arm64``.
+You can also query the available modules:
+
+.. code-block:: console
+
+    aqt list-qt windows desktop --modules 6.2.0
+
+
 When you decide to install Qt SDK version 6.2.0 for mingw v8.1
 
 .. code-block:: console
 
-    aqt install-qt windows desktop 6.2.0 win64_mingw81
+    aqt install-qt windows desktop 6.2.0 win64_mingw81 -m all
+ 
+The optional `-m all` argument installs all the modules available for Qt 6.2.0; you can leave it off if you don't want the modules.
```

Comment:
```suggestion
The optional `-m all` argument installs all the modules available for Qt 6.2.0; you can leave it off if you don't want those modules.
```

@ddalcino , I think using "those" instead of "the" makes more sense. We are talking about the modules from Qt, and we didn't mention one or more specific ones, so they are "far from us".

What do you think of it?

Reply:
I think it works either way, but I like your reasoning, and I agree with the change.

Reply:
OK, thanks for your reply.

@miurahr , please review [this suggestion](https://github.com/miurahr/aqtinstall/pull/377#discussion_r698934855).

Reply:
I agree with the change.